docs/manual/lean_tutorial/index.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
As we discussed in https://www.notion.so/Lean-tutorial-manual-2568fd40c8e380328f94d011dfac8f1d?source=copy_link, we probably would prefer to have a F* chapter that includes a quick start, a tutorial, etc

Rather than a 'F* quick start', 'F* tuto', etc. at top level

Can you fix that?
Also, plaintext `Fstar` doesn't read nicely, can you change the title to "F*"? (there is an option to do so while keeping full fstar in filenames, I don't recall exactly)